src/core/app_config.c

```diff
@@ -16,5 +16,9 @@ void srn_application_config_free(SrnApplicationConfig *cfg){
 }
 
 SrnRet srn_application_config_check(SrnApplicationConfig *cfg){
+    if (g_strcmp0(cfg->ui->window.chat_list_order, CHAT_LIST_ORDER_RECENT) != 0
+            && g_strcmp0(cfg->ui->window.chat_list_order, CHAT_LIST_ORDER_ALPHABET) != 0){
+        return SRN_ERR;
```

Comment:
You can return RET_ERR(xxx") to provide a more friendly error message.

Reply:
@SilverRainZ Should I update the PO files after adding an error message?

Reply:
It is fine, we can do this when we want to refresh our translation.